blocks/components/editable/index.js

```diff
@@ -154,6 +172,22 @@ export default class Editable extends wp.element.Component {
 		}
 	}
 
+	componentWillReceiveProps( nextProps ) {
+		forEach( nextProps.formats, ( state, format ) => {
+			const currentState = this.formats[ format ] || false;
+
+			if ( state !== currentState ) {
```

Comment:
>I'm not sure how that helps?

It's more a style consideration of avoiding [pyramid of doom](https://en.wikipedia.org/wiki/Pyramid_of_doom_(programming)) (and in other cases, [callback hell](http://callbackhell.com/)), where the problems become:

- Line length
- Maintaining block closings
  - Or in the case of the reader, making sense of separation between `if` and `else` blocks, when blocks are particularly long
- In the case of callbacks, having a good sense of independent function responsibilities

All of which are easily addressed by an emphasis on keeping code as shallow as possible by terminating early or deferring remainder of logic to separate function(s).

In this case, the code is not particularly difficult to make sense of. I only make the suggestion as I find it's almost always a better style to default to, especially if we expect to add more to these functions in the future (becomes less easy or obvious to flatten in the future).